src/db/build.ts

```diff
@@ -21,9 +21,7 @@ const buildDB = async () => {
   log('Appointments created');
   await Session.bulkCreate(Sessions);
   log('Sessions created');
-  if (process.env.BUILD) {
-    process.exit();
-  }
+  if (!(process.env.NODE_ENV === 'testing'))process.exit();
 };
 
```

Comment:
Seems we emerged a code that delete the call of `buildDB`, can u please add this 
```js
if (process.env.BUILD) {
  buildDB();
}
```